### ISSUE TITLE
No ACK from MQTT server in 10 seconds

### ISSUE THREAD

**Issue by zetkasprzyk** [hassio-addons/addon-emqx#4]
Home Assistant logs randomly ( I'm not able to corelate them with any particular event ) throw few errors saying 
"**No ACK from MQTT server in 10 seconds (mid ####)**” when using EMQX addon and default MQTT integration.
 It works as expected for few hours then those errors appear in HA's log and SOME mqtt entities become unavailable. 
Restarting the addon or HA gets those entities back. Until the next error. 
No errors when using Mosquito addon. 

Home Assistant 2022.9.6
Supervisor 2022.09.1
Operating System 9.0

EMQX version: 0.1.1 ( default settings ).

**Comment by Joeboyc2** [comment]
Im having a similar issue, and it seems pretty random, maybe there is a setting in EMQX that needs changing to a similar value mosquito uses.
Also had an issue after HA restart.
Ive had to move back to Mosquitto for the time being

**Comment by zetkasprzyk** [comment]
I'll try to do some debugging and see if anything consistent comes up in any logs. 
Let me know if you want me to check anything specific. 
Thank you!

**Comment by zetkasprzyk** [comment]
More than 24 hours have passed and no errors. Everything is working as expected. 
Steps I took:

 - Uninstalled Mosquito broker addon. Even when the addon was stopped, watchdog turned off, it was causing some weird behaviour with MQTT integration. with EMQX addon.
  
- Removed MQTT integration from HA.

- In EMQX created separate user just for HA and made it a superuser.

- In EMQX turned on ACL chache and increased ttl to 5 minutes.

- Restarted the whole system. 

- Added MQTT integration in HA, configured to use EMQX and authentication as that user created just for HA. 

- Restarted HA. 

Not sure if this issue was caused by EMQX or something else. Not sure if I fixed it definitely, but before that errors would appear after 1...4 hours. No errors so far. 
Most importantly - not sure which of those steps helped. But it's my production instance and I don't really have time to investigate more...
Hope it'll help.
